service/share/full_availability_test.go

```diff
@@ -35,11 +38,54 @@ func TestShareAvailableOverMocknet_Full(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	net := NewDAGNet(ctx, t)
-	_, root := net.RandFullService(16)
-	serv := net.CleanService()
+	net := NewTestDAGNet(ctx, t)
+	_, root := net.RandFullNode(32)
+	nd := net.FullNode()
 	net.ConnectAll()
 
-	err := serv.SharesAvailable(ctx, root)
+	err := nd.SharesAvailable(ctx, root)
+	assert.NoError(t, err)
+}
+
+func TestShareAvailable_FullOverLights(t *testing.T) {
+	ipld.RetrieveQuadrantTimeout = time.Millisecond * 200
+	DefaultSampleAmount = 20 // s
+
+	const (
+		origSquareSize = 16 // k
+		lightNodes     = 69 // c
```

Comment:
nice

Reply:
I updated this test and added more complex test scenarios. Check them out as well